pkg/policyexceptions/exception_controller.go

```diff
@@ -0,0 +1,118 @@
+package policyexceptions
+
```

Comment:
Please look at the controllers in the `/pkg/controllers` package, it should help you structure controller code better (if we agree we need one).